templates/readme.md

```diff
@@ -0,0 +1,61 @@
+# DOT Graph template
+
+```
+crane template -t path/to/graph.tmpl -o output/myGraph.dot
+```
+
+Generate a [DOT](http://en.wikipedia.org/wiki/DOT_(graph_description_language)) file representing the dependency graph. Bold nodes represent the containers declared in the config (as opposed to non-bold ones that are referenced in the config, but not defined). Targeted containers are highlighted with color borders. Solid edges represent links, dashed edges volumesFrom, and dotted edges net=container relations.
+
+# Systemd template
```

Comment:
Yes, but I think the idea should be that these are only examples. Crane shouldn't support them… I can add more templates in the `test_templates` folder for them.